packages/Search/README.MD

```diff
@@ -1,7 +1,11 @@
 
 <!-- start: Autogenerated - do not modify -->
 
+<<<<<<< HEAD
```

Comment:
@allison-c

Reply:
Fixed! Also fixed this file's name, was `.MD` before, should be `.md`, that's why the changes never got committed by script